From bde5aad1e9f7bfb6f8d5c60197797de4a315158d Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Thu, 14 Jun 2012 02:56:31 +0530
Subject: [PATCH 2/5] Revert "Always rec-connect if forced (ie. when Nepomuk
 server restarts)"

This reverts commit a43824a200ed76cef48a90bd3aee8547698b7c91.

This is done so that 4.8.x works with Soprano 2.7.6
---
 nepomuk/core/nepomukmainmodel.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nepomuk/core/nepomukmainmodel.cpp b/nepomuk/core/nepomukmainmodel.cpp
index ff742d7..be23b38 100644
--- a/nepomuk/core/nepomukmainmodel.cpp
+++ b/nepomuk/core/nepomukmainmodel.cpp
@@ -94,7 +94,7 @@ public:
 
         // we may get disconnected from the server but we don't want to try
         // to connect every time the model is requested
-        if ( forced || (!m_socketConnectFailed && !localSocketClient.isConnected()) ) {
+        if ( !m_socketConnectFailed && !localSocketClient.isConnected() ) {
             delete localSocketModel;
             localSocketModel = 0;
             QString socketName = KGlobal::dirs()->locateLocal( "socket", "nepomuk-socket" );
-- 
1.7.9.2

